src/app/_models/harvest-data.ts

```diff
@@ -4,9 +4,11 @@ export interface HarvestData {
   lastPreviewDate: string;
   lastPreviewRecords: number;
   lastPreviewRecordsReadyForViewing: boolean;
-
   firstPublishedDate: string;
+  isCurrentlyDepublished?: boolean;
+  lastDepublishedDate?: string;
   lastPublishedDate: string;
+  lastDepublishedRecords?: number;
```

Comment:
I don't know whether these are the right fields. Have you agreed them with Simon?

Reply:
As per our private conversation with Simon we're keeping the bottom 3 and changing the first to an enumeration.  I've renamed the variable to "currentPublicationStatus"